scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -170,17 +189,29 @@ def configure(self, instance):
             for r in self.regex:
                 self.patterns.append(re.compile(r))
             self.patterns_expected = len(self.patterns)
+        else:
+            self.state = "failed"
+            tc = self.instance.set_case_status_by_name(
+                self.id,
```

Comment:
why not use self.get_testcase_name() for this?

Reply:
good catch, I've forgot it here.